README.rst

```diff
@@ -1,2 +1,95 @@
-layered-config
+layered_config
 ==============
+
+A simple library to ease maintaining groups of similar configurations:
+   * Keep config files DRY like code.
+   * Allow environment variables to override configuration file values.
+   * `munch`_ your loaded configuration for easy field access.
+
+Instead of copy'n'paste-ing related configuration files for different situations,
+refactor the configuration sections and keys into separate smaller configuration files.
+Then use this library to combine those files in various ways to produce a single |ConfigParser| object,
+as if it had come from a single file.
+
+
+tl;dr Quick Start in 3 Easy Steps:
+----------------------------------
+
+1. Create some configs:
+~~~~~~~~~~~~~~~~~~~~~~~
+
+For this simple example, consider three config files:
+
+    | ``base.config`` - all the common stuff between your environments.
+    | ``staging.config`` - config info specific to your staging environment.
+    | ``demo.config`` - config info specific to your demo environment.
+
+Create a ``master.config`` file describing how to layer the simpler config files.
+The master config file's name is arbitrary.
+Your code needs to know its name, but this library doesn't care.
+
+``master.config`` file:
+
+.. code:: INI
+
+    [env1]
```

Comment:
perhaps these sections should be `staging` and `demo`. That makes it clear how they are related and match every current use case that I'm aware of.

Reply:
Thanks! I struggled with that because I wanted to show that the section name and the file names do not have to be related at all. Have an idea, let's see how it looks :-)